website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+In terms of the FLINT programming framework, there are internal and external modules. The internal modules tend to be generic utility modules, for example, for producing output tables. While external modules are regionally or nationally specific. By using modules and separating the generic and specific, FLINT’s framework remains highly flexible and new modules can be ‘plugged in’ as they are developed.
+
+### Pools
+
+A pool is a reservoir within which something can be stored and released. For example, a carbon pool is a reservoir from which carbon can be stored (sequestered and/or maintained) or emitted, such as debris. Within FLINT, each pool has attributed a value such as tonnes carbon, and at each time step, the FLINT can move stores from one pool to another using an operation. Here are the different pools present in FLINT.
+
+|![Diagram depicting how modules determine pools in FLINT.](../static/img/pools-module-case-study.png) |
+|:--:|
+| Diagram depicting how modules determine pools in FLINT. |
+
+The **Intergovernmental Panel on Climate Change** ([IPCC](https://www.ipcc.ch/about/)) is the United Nations body for assessing the science related to climate change. FLINT has been designed to follow the IPCC Good Practice Guidance and the evolving transparency framework under the Paris Agreement. The approach is solidly rooted in science and applies a carbon balance method to ensure the generated results are reliable.
+
+### Operations
+
+An operation is a process within FLINT that moves carbon stock between pools. Operations can reflect ongoing natural processes, such as growth, or specific events, whether natural or human-induced. Operations allow FLINT to track changes in carbon stock through time, including fluxes into and out of pools. For example, a harvest operation moves plant material to products and debris pools, whereas a wildfire operation moves plant material to debris and atmosphere pools.
+
+### Events
+
+Events are operations that occur intermittently (rather than for every time step in a simulation) resulting in the movement of carbon from one pool to another. Events include natural and anthropogenic events including: fire, harvesting, ploughing, and fertiliser application. These are coded for the FLINT as a module. Here is the relationship between Pools and Operations/ Events.
+
+|![Diagram depicting how FLINT calls modules to move carbon between pools.](../static/img/operations-module-case-study.png) |
+|:--:|
+| Diagram depicting how FLINT calls modules to move carbon between pools. |
```

Comment:
I would add more detail to the caption, explaining how to interpret the colours and simples. Work through a few examples.

> The first column says XX, and the first row say YY. We can therefore say that ZZZ